### ISSUE TITLE
localTime attribute does not take effect

### ISSUE THREAD

**Issue by BinitaBharati** [winstonjs/winston-daily-rotate-file#114]
Am using winston-daily-rotate-file (^1.7.2), even though localTime is set to true, timestamps are being logged into file in default UTC format.
Here's is the logger config:

``` javascript
 logger : new (winston.Logger)({
    transports: [
      new (winston.transports.DailyRotateFile)({
	      filename: 'arms-rest.log',
	      dirname: '/root/arms-rest2/arms-rest/logs/',
	      datePattern: 'yyyy-MM-dd.',
	      **localTime**: **true**,
	      prepend: true,
	      json: false,
	      maxDays: 2
  		})

    ]
  })
```

**Comment by mattberther** [comment]
What is the timezone of the computer that the application is running on? Any chance it is set to UTC?

**Comment by Superlukec** [comment]
I have the same problem. My computer is not set to UTC.

**Comment by mattberther** [comment]
Some long-overdue refactoring of the transport has occurred. A PR is available at #107. Also, a pre-release version has been published to npm. Please `npm install winston-daily-rotate-file@2.0.0-beta`. 

Note that localTime is no longer an option on that version, rather it uses the momentjs strings to display the date. More information is available here: http://momentjs.com/docs/#/displaying/

Please re-open this issue if you continue to see weird behavior with the 2.0.0 beta.

**Comment by BinitaBharati** [comment]
Was out for holidays. The time zone of the computer where the application is running is set to EST. Good to know, that a beta fix is provided.
